com.unity.render-pipelines.high-definition/Documentation~/Material-Type.md

```diff
@@ -38,7 +38,7 @@ Unity exposes different properties for your Material depending on the **Material
 | **Anisotropy**                        | Use the slider to set the direction of the anisotropy effect. Negative values make the effect vertical, and positive values make the effect horizontal. This stretches the specular highlights in the given direction.<br />This property only appears when you select **Anisotropy** from the **Material Type** drop-down. |
 | **Anisotropy Map**                    | Assign a Texture, with values from 0 to 1, that controls the strength of the anisotropy effect. HDRP only uses the red channel of this Texture to calculate the strength of the effect.<br />This property only appears when you select **Anisotropy** from the **Material Type** drop-down. |
 | **Iridescence Mask**                  | Assign a Texture, with values from 0 to 1, that controls the strength of the iridescence effect. A texel with a value of 1 corresponds to full strength, while those with a value of 0 disables the iridescence effect.<br />This property only appears when you select **Iridescence** from the **Material Type** drop-down. |
-| **Iridescence Layer Thickness map**   | Assign a Texture, with values from 0 to 1, that controls the thickness of the iridescence. This modifies the color of the effect.<br />This property only appears when you select **Iridescence** from the **Material Type** drop-down. |
+| **Iridescence Layer Thickness map**   | Assign a Texture, with values from 0 to 1, that controls the thickness of the thin iridescence layer over the material. This modifies the color of the effect. Unit is micrometer multiply by 3. A value of 1 will be remap to 3 micrometer or 3000 nanometer.<br />This property only appears when you select **Iridescence** from the **Material Type** drop-down. |
```

Comment:
Unit is micrometer multipl**ied** by 3. A value of 1 **is remapped** to 3 micrometer**s** or 3000 nanometer**s**